.github/workflows/create-release.yml

```diff
@@ -45,6 +45,12 @@ jobs:
       GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     steps:
       - uses: actions/checkout@v3
+      - name: Install libssl-dev
+        if: matrix.target == 'x86_64-unknown-linux-gnu'
+        run: sudo apt-get install libssl-dev pkg-config
```

Comment:
You can move the command to a "build-command" entry in the strategy matrix. Then here, you'd just need to invoke the command (`$(${{ matrix.build-command }})`?).